index.js

```diff
@@ -13,7 +13,7 @@ const plugins = require("./plugins");
 const logoTemplate = require("./templates/logo.handlebars");
 
 // Chrome Accessibility Developer Tools - required once as a global
-require("script!./node_modules/accessibility-developer-tools/dist/js/axs_testing.js");
+require("script-loader!./node_modules/accessibility-developer-tools/dist/js/axs_testing.js");
```

Comment:
That's annoying that they don't have a UMD version.

Reply:
There might be a better way, now. But that'll have to be a different change. :)